PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.core/src/com/microsoft/azuretools/core/ui/SignInDialog.java

```diff
@@ -321,27 +333,26 @@ private void signInAsync(final DCAuthManager dcAuthManager) throws InvocationTar
 
     private void doSignOut() {
         accountEmail = null;
-        DCAuthManager.getInstance().signOut();
+        // AuthMethod.AD is deprecated.
+        getAuthMethodManager().getAdAuthManagerByAuthMethod(AuthMethod.DC).signOut();
```

Comment:
Shall we use `getAuthMethodManager().signOut();` here?

Reply:
Looks like the current method is only for DC auth.